miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/SampleRestController.java

```diff
@@ -317,14 +321,14 @@ public SampleDto updateSample(@PathVariable("id") Long id, @RequestBody SampleDt
     return matches.stream().map(identity -> Dtos.asDto(identity)).collect(Collectors.toSet());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)